build.sbt

```diff
@@ -7,13 +7,16 @@ name := "podcasts-analytics-lambda"
 
 lazy val root = (project in file(".")).enablePlugins(JavaAppPackaging, RiffRaffArtifact)
 
+val log4jVersion = "2.11.0"
+
 libraryDependencies ++= Seq(
   "org.joda" % "joda-convert" % "1.8.1",
-  "com.typesafe.scala-logging" %% "scala-logging" % "3.9.0",
+  "org.apache.logging.log4j" %% "log4j-api-scala" % "11.0",
+  "com.amazonaws" % "aws-lambda-java-log4j2" % "1.1.0",
```

Comment:
Can all the AWS dependency share the same version number?

Reply:
indeed, we've now get 3 different versions!

Reply:
actually, it appears these libraries all follow their own versioning